README.md

```diff
@@ -59,6 +59,16 @@ than mock (above).
 
     obal scratch PACKAGE
 
+However, `scratch` will not work for nightly packages (i.e. - packages that
+don't have sources tracked in foreman-packages). The process for these is a bit
+different. Build the source files externally, then point `obal nightly` at
+them.
+
+    obal nightly foreman-installer \
+        --scratch \
+        --source ~/workspace/foreman-installer/pkg/foreman-installer-1.20.0-develop.tar.bz2 \
+        --githash 3ff078d12560c3f7ec55dad6bc70dd87dd7855a0
```

Comment:
Is this part required? Might be worth mentioning acquiring this from somewhere if it is especially if I got my source from somewhere else.

Reply:
the git has is require. it's needed for the `<timestamp>git<hash>` string that appended to the release.